plugins/source/aws/client/services.go

```diff
@@ -375,3 +256,249 @@ type Services struct {
+	case "neptune":
+		s.Neptune = neptune.NewFromConfig(c)
+	case "networkfirewall":
+		s.Networkfirewall = networkfirewall.NewFromConfig(c)
+	case "networkmanager":
+		s.Networkmanager = networkmanager.NewFromConfig(c)
+	case "organizations":
+		s.Organizations = organizations.NewFromConfig(c)
+	case "qldb":
+		s.Qldb = qldb.NewFromConfig(c)
+	case "quicksight":
+		s.Quicksight = quicksight.NewFromConfig(c)
+	case "ram":
+		s.Ram = ram.NewFromConfig(c)
+	case "rds":
+		s.Rds = rds.NewFromConfig(c)
+	case "redshift":
+		s.Redshift = redshift.NewFromConfig(c)
+	case "resiliencehub":
+		s.Resiliencehub = resiliencehub.NewFromConfig(c)
+	case "resourcegroups":
+		s.Resourcegroups = resourcegroups.NewFromConfig(c)
+	case "route53":
+		s.Route53 = route53.NewFromConfig(c)
+	case "route53domains":
+		s.Route53domains = route53domains.NewFromConfig(c)
+	case "route53recoverycontrolconfig":
+		s.Route53recoverycontrolconfig = route53recoverycontrolconfig.NewFromConfig(c)
+	case "route53recoveryreadiness":
+		s.Route53recoveryreadiness = route53recoveryreadiness.NewFromConfig(c)
+	case "route53resolver":
+		s.Route53resolver = route53resolver.NewFromConfig(c)
+	case "s3":
+		s.S3 = s3.NewFromConfig(c)
+	case "s3control":
+		s.S3control = s3control.NewFromConfig(c)
+	case "sagemaker":
+		s.Sagemaker = sagemaker.NewFromConfig(c)
+	case "savingsplans":
+		s.Savingsplans = savingsplans.NewFromConfig(c)
+	case "scheduler":
+		s.Scheduler = scheduler.NewFromConfig(c)
+	case "secretsmanager":
+		s.Secretsmanager = secretsmanager.NewFromConfig(c)
+	case "securityhub":
+		s.Securityhub = securityhub.NewFromConfig(c)
+	case "servicecatalog":
+		s.Servicecatalog = servicecatalog.NewFromConfig(c)
+	case "servicecatalogappregistry":
+		s.Servicecatalogappregistry = servicecatalogappregistry.NewFromConfig(c)
+	case "servicediscovery":
+		s.Servicediscovery = servicediscovery.NewFromConfig(c)
+	case "servicequotas":
+		s.Servicequotas = servicequotas.NewFromConfig(c)
+	case "ses":
+		s.Ses = ses.NewFromConfig(c)
+	case "sesv2":
+		s.Sesv2 = sesv2.NewFromConfig(c)
+	case "sfn":
+		s.Sfn = sfn.NewFromConfig(c)
+	case "shield":
+		s.Shield = shield.NewFromConfig(c)
+	case "signer":
+		s.Signer = signer.NewFromConfig(c)
+	case "sns":
+		s.Sns = sns.NewFromConfig(c)
+	case "sqs":
+		s.Sqs = sqs.NewFromConfig(c)
+	case "ssm":
+		s.Ssm = ssm.NewFromConfig(c)
+	case "ssoadmin":
+		s.Ssoadmin = ssoadmin.NewFromConfig(c)
+	case "support":
+		s.Support = support.NewFromConfig(c)
+	case "timestreamwrite":
+		s.Timestreamwrite = timestreamwrite.NewFromConfig(c)
+	case "transfer":
+		s.Transfer = transfer.NewFromConfig(c)
+	case "waf":
+		s.Waf = waf.NewFromConfig(c)
+	case "wafv2":
+		s.Wafv2 = wafv2.NewFromConfig(c)
+	case "wafregional":
+		s.Wafregional = wafregional.NewFromConfig(c)
+	case "wellarchitected":
+		s.Wellarchitected = wellarchitected.NewFromConfig(c)
+	case "workspaces":
+		s.Workspaces = workspaces.NewFromConfig(c)
+	case "xray":
+		s.Xray = xray.NewFromConfig(c)
+	}
```

Comment:
Let's add a default case here that panics if the function is called with an unrecognized service

Reply:
Done in https://github.com/cloudquery/cloudquery/pull/13059/commits/a08568ce98f8c0f120c6fd44ed7d0ebdae6fc8aa